docs/cookbook/configuration/disabling-shop-admin-api.rst

```diff
@@ -82,6 +87,18 @@ The part that has to be removed from this file is shown below:
     #                success_handler: sylius.handler.shop_user_logout
     #            anonymous: true
 
+    access_control:
```

Comment:
I think the same here, and for `security.firewalls` I think they all should have `~` at the end... @lchrusciel could you check that?